demo-projects/meetup/site/components/Talks.js

```diff
@@ -28,7 +27,9 @@ const Talks = ({ talks }) => {
               <H5 as="h3">{talk.name}</H5>
             </div>
             <Content>
-              {talk.description ? <Html markup={talk.description} /> : null}
+              {talk.description ? (
+                <Html markup={talk.description.replace(/style=".*?"/g, '')} />
```

Comment:
Is this something that the `Html` component should handle?

Reply:
This regexp is too greedy, it'll cause issues.

A quick and dirty fix would at least be this: `/style="[^"]*"/g`

What's the problem being guarded against here?

Reply:
the problem is that many of the descriptions for talks and events have custom styling that clash with the designs.